lib/bootstrap_form/inputs/check_box.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
```

Comment:
Can you please just confirm that you didn't change any of the code in this file, `radio_button.rb`, the code that's left in `form_builder.rb`, and anywhere else where you were copying and pasting, other than the bare minimum you needed to change to move the code?

Reply:
Confirmed. The sole change outside of this was in removing the method_chain_alias code and implementing the cleaner equivalent in `base.rb` as `bootstrap_alias`

Reply:
Thanks!